### DISCUSSION TITLE
有近視的人比較聰明

### DISCUSSION THREAD

**Discussion by roma0323** [1101-GS4719-C/Python-Community-GS4719-C#1550] [General]
英文有份研究發現高智商的人，近視機率會高出32%

另根據調查，台灣年輕人口近視比例為77%，其他和近視有關的數據，也都能領先全球，所以說在座的各位，都是聰明又耐操充滿了競爭力呢？
但先別急，因為分析採用的基因庫來自歐洲，所以台灣是不是也這樣，不能直接下定論，但有趣的是，或許是受到升學文化影響，台灣人大多近視源自不良的用眼習慣
但歐美則相反，他們普遍相信是先天基因決定

我認為兩種觀點各對一半
會不會近視，就像一個人會不會成功
先天基因和後天努力，勢必都會造成影響

**Comment by poweichen00** [comment]
近視的發生有多因素所造成，包括遺傳、環境、生活習慣等，無論是遺傳因素還是後天努力，都可能對視力產生影響，重要的是保持良好的生活習慣，包括適度用眼、注意休息、保持適當的閱讀姿勢等，能保持視力健康。

**Comment by lyien1021** [comment]
怎麼覺得這個標題下的怪怪的，應該是高智商份子近視的機率比別人高，不等於近視的人比較聰明的意思吧🤔

**Comment by larry40305** [comment]
我認為這個問題的成因很可能是多元的，既包括遺傳基因的影響，也包括生活習慣和環境因素的影響。近視和成功一樣，可能受到多種因素的交互影響。

**Comment by aa103967622** [comment]
近視的原因可能是用眼過度，過度操勞你的眼睛，未適度休息，雖然這也可能代表你在學科上的努力，但是我們還是要擁有良好的用眼習慣，每用眼30分鐘，就休息個十分鐘，看看遠方，這樣能確保你擁有良好視力，能繼續幫助你在學業上進步。

**Comment by zhiron544** [comment]
文章內容有趣，但標題有誤導之嫌。近視和智商可能有關聯，但不是必然。保持良好的用眼習慣，才是最重要的。

**Comment by Alex11392** [comment]
好有趣的內容，雖然比較聰明好像很棒，但我還是比較想要健康的眼睛ㄚㄚ!

**Comment by lin157644** [comment]
台灣或是一些亞洲國家注重學童學業表現，從小就花上許多時間盯著書籍電腦，然而兒童眼睛還在發於容易因此導致近視，要說近視與智商表現有關也是很合理，統計上正相關不等於因果關係，但是近視的我選擇相信兩者之間是有關係的:smile:。
